src/commands/new-proj.ts

```diff
@@ -20,20 +28,86 @@ async function action() {
     if (nameResult.name) {
       projName= nameResult.name;
     } else {
-      throw new Error("ERROR: No name was given for the project");
+      throw new Error("No name was given for the project");
+    }
+
+    const typeResult = await this.prompt({
+      type : 'list',
+      name : 'type',
+      choices : ["Quarkus", "React", "No framework"],
+      message : "Framework for the project: "
+    });
+
+    if (typeResult.name) {
+      projType= typeResult.name;
+    } else {
+      throw new Error("No type was given for the project");
+    }
+
+    const pathResult = await this.prompt({
+      type : 'input',
+      name : 'path',
+      message : "Set a path where to initiate repository, leave empty for default: "
+    });
+
+    if (pathResult?.path) {
+      givenPath = pathResult.path;
     }
   } catch (err) {
-    throw new Error(err);
+    throw new Error("Error: " + err);
+  }
+
+  givenPath = await PathUtils.fixPath(givenPath);
+  const folderPath : string = PathUtils.outerFolder(givenPath, projName);
+  const repoPath : string = PathUtils.repoFolder(givenPath, projName);
+
+  if (projType == "Quarkus") {
+    //To do: Add quarkus
+    this.log("Creating Quarkus project - please wait...");
+  }
+
+  if (projType == "React") {
+    //To do: Add React
+    this.log("Creating react project - please wait...");
+  }
+
+  if (projType == "No framework") {
+    this.log("Creating project - please wait...");
+    const cmds : string[] = await CreateDefault(projName, folderPath, repoPath);
+    execSync(cmds[0]);
+    execSync(cmds[1], {cwd : `.${path.sep}resources`});
+    execSync(cmds[2], {cwd : `.${path.sep}resources`});
   }
 
   try {
-    await vorpal
-    .use(newRepo)
-    .execSync("new-repo");
-  } catch(err) {
-    throw new Error("Encountered an error while creating repository: " + err);
+    const repoResult = await this.prompt({
+      type : 'list',
+      name : 'repo',
+      choices : ["Yes", "No"],
+      message : "Create GitHub repository for the project?: "
+    });
+
+    if (repoResult.name) {
+      if (repoResult.name == "Yes") {
```

Comment:
probably not a needed change as the prompt might be unneeded but
```suggestion
    if (repoResult.repo) {
      if (repoResult.repo == "Yes") {
```

Reply:
prompt ended up being unnecessary